### PR TITLE
EZEE-3213: Added verification if the user has section/view permission

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZEE-3213](https://issues.ibexa.co/browse/EZEE-3213)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Verification of permission according to other forms in our platform

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Review by webhdx — CHANGES_REQUESTED** [review]
This is not a reliable way to check permissions. Simply use `PermissionResolver` and inject additional variable to the Twig template.

File: src/bundle/Controller/TrashController.php [hunk 1]
```diff
@@ -165,6 +165,7 @@ public function listAction(Request $request): Response
             'can_delete' => $this->isGranted(new Attribute('content', 'remove')),
             'can_restore' => $this->isGranted(new Attribute('content', 'restore')),
             'can_cleantrash' => $this->isGranted(new Attribute('content', 'cleantrash')),
+            'can_section' => $this->isGranted(new Attribute('section', 'view')),
```
Comment (webhdx) [inline src/bundle/Controller/TrashController.php]: Please use proper name: `can_view_section`.

**Review by webhdx — CHANGES_REQUESTED** [review]

**Review by webhdx — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by mikadamczyk — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by kacper-wieczorek-ibexa — APPROVED** [review]
QA - approved. Tested on 3.3.12 experience, commerce.
It is possible to open trash as editor.

**Comment by lserwatka** [comment]
@mateuszdebinski you must merge it to ibexa/admin-ui:main

**Pull request merged**